exercises/L-functions-nested/exercise.js

```diff
@@ -1,5 +1,19 @@
-var mentor1 = "Daniel";
-var mentor2 = "Irina";
-var mentor3 = "Mimi";
-var mentor4 = "Rob";
-var mentor5 = "Yohannes";
+
+function greetings(mentor) {
+let mentor1 = "Daniel";
+let mentor2 = "Irina";
+let mentor3 = "Mimi";
+let mentor4 = "Rob";
+let mentor5 = "Yohannes";
```

Comment:
These variables don't do anything here, because you are passing the strings directly into the mentor parameter.

Reply:
Corrected, thank you.